CalibTracker/SiStripCommon/test/MakeCalibrationTrees/produceCalibrationTree_template_cfg.py

```diff
@@ -1,4 +1,70 @@
 import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+###################################################################
+# Setup 'standard' options
+###################################################################
+options = VarParsing.VarParsing()
+
+options.register('conditionGT',
+                 "76X_dataRun2_v15",
```

Comment:
@mmusich

All the template uses the symbolic GT from autoconf.py.

> On 07 Mar 2016, at 09:47, Marco Musich <notifications@github.com> wrote:
> 
> In CalibTracker/SiStripCommon/test/MakeCalibrationTrees/produceCalibrationTree_template_cfg.py <https://github.com/cms-sw/cmssw/pull/13609#discussion_r55174962>:
> 
> > @@ -1,4 +1,70 @@
> >  import FWCore.ParameterSet.Config as cms
> > +import FWCore.ParameterSet.VarParsing as VarParsing
> > +
> > +###################################################################
> > +# Setup 'standard' options
> > +###################################################################
> > +options = VarParsing.VarParsing()
> > +
> > +options.register('conditionGT',
> > +                 "76X_dataRun2_v15",
> since this is a template, can the GT be made symbolic?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/cmssw/pull/13609/files#r55174962>.
>